lib/opensusebasetest.pm

```diff
@@ -806,6 +806,18 @@ sub post_fail_hook {
         }
     }
 
+    if (get_var('FULL_LVM_ENCRYPT') && get_var('LVM_THIN_LV')) {
+        my $self = shift;
+        select_console 'root-console';
+        my $lvmdump_regex = qr{/root/lvmdump-.*?-\d+\.tgz};
+        my $out           = script_output 'lvmdump';
+        if ($out =~ /(?<lvmdump_gzip>$lvmdump_regex)/) {
+            upload_logs "$+{lvmdump_gzip}";
+        }
+        $self->save_and_upload_log('lvm dumpconfig', '/tmp/lvm_dumpconf.out');
+        $self->SUPER::post_fail_hook;
```

Comment:
I believe it might even create infinite loop, so let's remove this line.

Reply:
i removed it but i cant see what would cause infinite loop in this case. 
i think this will call the post_fail_hook from basetest which will try cleanup if it is not cleaned up before

Reply:
You are right, I haven't check from where we inherit, but anyway, we don't need it here.